base/data/Campaign.js

```diff
@@ -242,19 +225,21 @@ Campaign.filterNonServedAds = (ads, showNonServed) => {
  * Get a list of adverts that the impact hub will hide for this campaign
  * @param {Campaign} topCampaign the subject campaign
  * @param {?Campaign[]} campaigns any other campaigns with data to use (for advertisers or agencies)
- * @returns {Advert[]} adverts to hide
+ * @param {?KStatus} status
+ * @returns PromiseValue(Advert[]) adverts to hide - returns null if no hide adverts
  */
-Campaign.advertsToHide = (topCampaign, campaigns) => {
+Campaign.advertsToHide = (topCampaign, campaigns, status=KStatus.DRAFT) => {
```

Comment:
Why do we want to fetch the adverts to hide? I.e. why isn't it enough to know which IDs to hide?

Reply:
Currently this is only used for portal controls, and not Impact Hub, which in itself just gets the ID list (via the Campaign.advertsToShow method). It's necessary for the portal controls however to have the hidden ad objects themselves, so they can be viewed like other ads in the controls as a list, and unhidden or modified as other ads can be from those controls. This could be done inside the portal controls themselves, but I decided to try and keep any code relating to fetching inside Campaign if I could, so it's less clutter and reusable if need be

Reply:
Please add a brief version of this to the doc, e.g. "Use case: for Portal, so the controls for hidden ad objects can show ad info"